### DISCUSSION TITLE
[V3] Modelable nested component does trigger parent update hooks

### DISCUSSION THREAD

**Discussion by CarlEdmondson** [livewire/livewire#6313] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.8

### Which PHP version are you using?

PHP 8.2

### Description

If I have a property bound to a nested child component using the _wire:model_ attribute, then when the child component updates the property it doesn't trigger any of the update lifecycle hooks on the parent component.

### Steps To Reproduce

_Parent Component_

    use Livewire\Component;
    class Parent extends Component
    {
        public $foo = "bar";
        
        public function mount()
        {
            //
        }
        
        public function render()
        {
            return view('parent');
        }
        
        public function updatedFoo()
        {
            // This is NOT triggered.
            dd('triggered'); 
        }
        
        public function updated()
        {
            // This is also NOT triggered.
            dd('triggered'); 
        }

        public function test()
        {
            // When called this will dump the updated foo value from the child component.
            dd('triggered', $this->foo); 
        }
    }

_Parent Component View_

    <div>
        <livewire:child wire:model="foo" />
        <button wire:click="test">Test</button>
    </div>

_Child Component_

    use Livewire\Attributes\Modelable;
    use Livewire\Component;
    class Child extends Component
    {
        #[Modelable]
        public $value = "";
        
        public function mount()
        {
            //
        }
        
        public function render()
        {
            return view('child');
        }
    }

_Child Component View_

    <div>
        <input wire:model.live="value" />
    </div>

If you change the value of the input in the child component then neither the _updatedFoo_ or the _updated_ methods on the parent component are called. However if you then call the _test_ method on the parent component by clicking the button then the value that is dumped is the updated value from the child component. So it seems like the value is being updated, but just not triggering the lifecycle hooks.

### Expected Outcome

I would expect that if the child component property is updated then it should cascade up to the parent and trigger the appropriate lifecycle hooks on the parent component.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by wizzymore** [comment]
I can confirm that the parent will only get the new state only when it will be triggered to re-render

If you do something like this in the child:
```php
    public function updated()
    {
        $this->dispatch('input');
    }
```

And this in the parent:
```php
    protected function getListeners()
    {
        return [
            'input' => '$refresh',
        ];
    }
```

I can confirm that it works as excepted, but doesn't it beat the point?
